installer/install.iss

```diff
@@ -57,7 +57,7 @@ DisableDirPage=auto
 DefaultGroupName={#APP_NAME}
```

Comment:
Yes, I see that the GPL file was moved. I just wanted this to be in a separate commit, with the necessaru change to `installer/release.sh`, leaving all `msi` stuff alone. The move is logically separate from, and preparatory to, the rest of this commit's changes.